src/main/java/com/faforever/client/map/MapVaultController.java

```diff
@@ -41,6 +42,7 @@
   private final EventBus eventBus;
 
   private MapDetailController mapDetailController;
+  private int recommendedShowRoomPageCount = 1;
```

Comment:
init with 0

Reply:
no better make it an integer, so it can be null. Let's make it so the vault does not break if something is misconfigured in the remote preferences

Reply:
Yeah I can make it null and add a null check but will still have to check for zero too since random.nextInt needs positive